test/src/index.test.ts

```diff
@@ -330,6 +330,21 @@ test('bladeSdk.contractCallFunction', async () => {
         expect(result.error.reason.includes("CONTRACT_REVERT_EXECUTED")).toEqual(true);
     }
 
+    try {
+        // fail on wrong function params (CONTRACT_REVERT_EXECUTED) with error_message
+        params = new ParametersBuilder()
+        result = await bladeSdk.contractCallFunction(contractId, "revert_fnc", params, accountId, privateKey, 1000000, true, completionKey);
+        checkResult(result);
```

Comment:
we should check that contractCallFunction throws error. replace with

```
expect("Code should not reach here").toBeNull();
```

Reply:
Fixed ✅